### ISSUE TITLE
Ditch "alwaysPull", bosco pull should pull docker images, not bosco run

### ISSUE THREAD

**Issue by geophree** [tes/bosco#54]
We shouldn't be pulling docker images every time we try to run them.  Bosco pull (and morning) should leave you with a consistent (+/- a few minutes) view of the project, so we don't want run to be pulling in the latest anything.  If something is out of date, people should be running pull/morning to bring it up to date.  This has the added benefit of making bosco run (for non-locally-built) docker repos much faster.

Thoughts?

**Comment by codeaholics** [comment]
I don't understand the issue. Can you explain in more detail? Perhaps with examples? I added the "latest" stuff (I think) to make sure local and development were behaving the same.

**Comment by geophree** [comment]
I'm talking about "alwaysPull", and that was added because once we downloaded an image, it was stuck on that one, and we were getting out of date.  Docker thought it had the latest, so it didn't try to pull again.

What I'm saying is that there's a better way to solve that issue.  We should put the "download latest image" part into "bosco pull", because it belongs there more than in "bosco run".  "pull" updates you to the latest, "run" runs what you have.
